config/ortho_bored.yml

```diff
@@ -80,40 +82,46 @@ base_layout:
       - ["3", "ℓ", "³", "№", "₃", "∧"]
       - ["4", "»", "›", "", "♀", "⊥"]
       - ["5", "«", "‹", "·", "♂", "∡"]
+      - [""]
+      - [""]
       - ["6", "$", "¢", "£", "⚥", "∥"]
       - ["7", "€", "¥", "¤", "ϰ", "→"]
       - ["8", "„", "‚", "⇥", "⟨", "∞"]
       - ["9", "“", "‘", "/", "⟩", "∝"]
-      - ["0", "”", "’", "*", "₀", "∅"]
-      - ["-", "—", "-", "‑", "­", ""]
+      - ["0", "”", "`", "*", "₀", "∅"]
+      - ["-", "—", "´", "‑", "­", ""]
```

Comment:
Is this in line with the "standard" neo2 map? In the standard keymap there were such symbols on one of the keys. I deleted them for lack of a fitting spot on the ortho keyboard maybe there is one on the moonlander (it has more keys). Otherwise, I would prefer staying in the "standard" neo2 layer configuration for the keys that are available.

Reply:
No, it is not in line with standard neo2 layers. There is no good spot on layer one of Moonlander either though. In an effort to avoid failing to cover (e.g. french) parts of the text body, I thought putting them on a higher layer might be an acceptable trade-off—and as a matter of fact, my personal keymap puts them on a higher level. I revert the changes for now, but somehow the issue should be addressed.

Reply:
It is not supported to generate "composed" symbols (e.g. e with French accent) anyway. Only symbols that appear as they are in the config can be mapped.
If one would like to support them, they wod have to appear in a separate layer (like an accented layer) od the corpus would need to take care of it.